packages/story-editor/src/utils/getMediaBaseColor.js

```diff
@@ -46,16 +47,15 @@ function getImgNodeKey(elementId) {
   return `${IMG_NODE}_${elementId}`;
 }
 
-export function getMediaBaseColor(resource, onBaseColor) {
-  const { type, src, poster } = resource;
-  setOrCreateImage({
-    src: type === 'video' ? poster : src,
+export function getMediaBaseColor(src) {
+  if (!src) {
+    return Promise.reject(new Error('No source to image'));
+  }
```

Comment:
Which scenario is this for? Where is this error caught? Why not a simple fallback?

Reply:
Sometime earlier when a video is created, a poster is not generated, meaning the source passed here might be empty. I believe I have got all the place this might happen, but just be safe, this needs to error, so try catches can work.

Reply:
Is this message ever surfaced to the user?

Reply:
Nope